src/web-worker/amp-worker.js

```diff
@@ -29,6 +29,38 @@ const TAG = 'web-worker';
 let PendingMessageDef;
 
 /**
+ * Error that can be passed through web worker
+ * @typedef {{
+ *   message: string,
+ *   stack: string,
+ * }}
+ */
+export let AmpWorkerErrorDef;
```

Comment:
> How does this create a dependency on bind-evaluator?

I meant `bind-evaluator` shouldn't depend on `amp-worker`.

> Also, what happens if/when more functionality depends on the worker. This prevents everyone who uses the worker from rolling their own error type.

I'm not so sure that future use cases will need to return `Error` objects. `bind-impl`'s usage of `reportError` is pretty particular.